sycl/doc/extensions/supported/sycl_ext_intel_esimd/sycl_ext_intel_esimd_functions.md

```diff
@@ -339,7 +339,36 @@ template <typename T, int N, int VS = 1, typename OffsetSimdViewT, typename Prop
 The parameter `byte_offset` is a vector of any integral type elements for `(usm-ga-*)`, 32-bit integer elements for `(lacc-ga-*)` and `(slm-ga-*)`, any integral type integer elements for `(acc-ga-*)` in [stateless](#statelessstateful-memory-mode) mode(default),
 and up-to-32-bit integer elements for `(acc-ga-*)` in [stateful](#statelessstateful-memory-mode) mode.  
 The optional parameter `pred` provides a `simd_mask`. If some element in `pred` is zero, then the load of the corresponding memory location is skipped and the element of the result is copied from `pass_thru` (if it is passed) or it is undefined (if `pass_thru` is omitted).  
-The optional [compile-time properties](#compile-time-properties) list `props` may specify `alignment` and/or `cache-hints`. The cache-hints are ignored for `(lacc-*)` and `(slm-*)` functions.
+The optional [compile-time properties](#compile-time-properties) list `props` may specify `alignment` and/or `cache-hints`. The cache-hints are ignored for `(lacc-*)` and `(slm-*)` functions.  
+The template parameter `N` can be any positive number.  
+The optional template parameter `VS` must be one of `{1, 2, 3, 4, 8, 16, 32, 64}` values. It specifies how many conseсutive elements are loaded per each element in `byte_offsets`.   
+### Example
+```C++
+simd<int64_t, 4> offsets(0, 100); // 0, 100, 200, 300 - offsets in bytes
+// loads and returns a vector {ptr[0], ptr[100/4], ptr[200/4], ptr[300/4]};
+simd<float, 4> vec4 = gather<float, 4>(ptr, offsets);
+
+// VS = 2, loads and returns a vector {ptr[0], ptr[100/4],     ptr[200/4],     ptr[300/4],
+//                                     ptr[1], ptr[100/4 + 1], ptr[200/4 + 1], ptr[300/4 + 1]};
+simd<float, 8> vec8 = gather<float, 8, 2>(ptr, offsets);
+```
+
+### Restrictions
+| `Function` | `Condition` | Required/supported Intel GPU |
```

Comment:
```suggestion
| `Function` | `Condition` | Required/supported Intel GPU |
```

Nit: I would pick either Required or supported instead of saying both

Reply:
It is copy/paste from other place.
If change this, then I'd do it in upcoming similar updates for atomic_update()/prefetch() + few older places.

Reply:
Thank you for quick code-review!

Reply:
The comment is going to be resolved in this PR: https://github.com/intel/llvm/pull/13202

Reply:
ah ok yeah i would definitely prefer it to be consistent everywhere, thanks for the new PR